From: Martin Michlmayr <tbm@cyrius.com>
Date: Sat, 19 Jan 2008 18:25:02 +0000
Subject: [mips] Disable Advansys
Forwarded: http://thread.gmane.org/gmane.linux.scsi/57291

Florian Lohoff <flo@rfc822.org> reports the following build failure on IP32:

  MODPOST 552 modules
ERROR: "free_dma" [drivers/scsi/advansys.ko] undefined!
make[5]: *** [__modpost] Error 1

But report:
http://www.mail-archive.com/linux-scsi@vger.kernel.org/msg12773.html

--- a/drivers/scsi/Kconfig
+++ b/drivers/scsi/Kconfig
@@ -505,6 +505,7 @@ config SCSI_ADVANSYS
 	tristate "AdvanSys SCSI support"
 	depends on SCSI
 	depends on ISA || EISA || PCI
+	depends on !MIPS || BROKEN
 	help
 	  This is a driver for all SCSI host adapters manufactured by
 	  AdvanSys. It is documented in the kernel source in
